src/cli/restore/sharepoint.go

```diff
@@ -64,6 +66,16 @@ func addSharePointCommands(cmd *cobra.Command) *cobra.Command {
 			utils.LibraryItemFN, nil,
 			"Restore library items by file name or ID")
 
+		fs.StringSliceVar(
+			&listPaths,
+			utils.ListFN, nil,
+			"Restore list items by SharePoint path")
+
+		fs.StringSliceVar(
+			&listItems,
+			utils.ListItemFN, nil,
+			"Restore list items by file name or ID")
```

Comment:
List items don't have file names, do they?
```suggestion
			"Restore list items by ID")
```